### ISSUE TITLE
Upgrade to PHP 7.1

### ISSUE THREAD

**Issue by soleo** [karllhughes/laravel-php-7#3]
Since PHP 7.1 is out, this should be upgraded as well

**Comment by karllhughes** [comment]
Agreed. I haven't installed PHP 7.1 yet, but I'm making a note to upgrade in January. In the meantime, we could probably just update the Travis CI file to run the tests in 7.1.

Should we require 7.1 or just make it optional?
